### PR TITLE
Feature/i18n

### PR DESCRIPTION
**Pull request by abyssparanoia**
## Proposed Changes

- added i18n packages
- added sample code for translation
- document update

### CODE REVIEW

File: packages/web/public/locales/en/PossessionOutline.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "totalRewards": "Total Reward" }
```
Comment (abyssparanoia) [inline packages/web/public/locales/en/PossessionOutline.json:1]: auto generated file

File: packages/i18n/src/locales/PossessionOutline.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+export const POSSESSION_OUTLINE_NAMESPACE = 'PossessionOutline'
```
Comment (k3nt0w) [inline packages/i18n/src/locales/PossessionOutline.ts:1]: Good Design 👏

File: packages/i18n/src/locales/run.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { promises as fs } from 'fs'
+import { possessionOutlineTranslationEN, possessionOutlineTranslationJP } from './PossessionOutline'
+
+const basePath = `${__dirname}/../../../web/public/locales`
```
Comment (k3nt0w) [inline packages/i18n/src/locales/run.ts:4]: I think it's better to use the path library to make it work safely on windows.

```suggestion
const basePath = path.join(__dirname, '../../..', 'web/public/locales')
```

**Comment by aggre** [comment]
@abyssparanoia I thought there was no need for environment variables anymore, so I deleted `.envrc` and wrote the URL in TypeScript. Revert if this change is a problem. https://github.com/dev-protocol/stakes.social/pull/304/commits/be3f6ddc1dcd844085511f0cf5cfc44010d2872a

**Comment by aggre** [comment]
I merge this!

**Pull request merged**